src/main/java/com/mg105/entities/GiveTutorial.java

```diff
@@ -7,6 +7,12 @@ public class GiveTutorial {
     private boolean attacked = false;
     private boolean usedItem = false;
 
+    public GiveTutorial(boolean moved, boolean attacked, boolean usedItem){
```

Comment:
why not just have these variables set to false by default? unless there's a scenario in which the user can redo certain parts of the tutorial I guess. I assumed the tutorial would be a one time thing